tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -1000,7 +999,7 @@ tensorflow::Status ConvertConv2DHelper(
     const std::vector<TRT_TensorOrWeights>& inputs,
     std::vector<TRT_TensorOrWeights>* outputs,
     int group  // group ==0 specifies depthwise conv
-) {
+    ) {
```

Comment:
Nit: please merge this line with the previous line. For the comment, either use `/**/` or put it after `) {`.